docs/example_usage/breeds_datasets.rst

```diff
@@ -0,0 +1,288 @@
+
+- :samp:`level`: Level in the hierarchy (in terms of distance from the
+  root node) at which to define superclasses.
+- :samp:`Nsubclasses`: Controls the minimum number of subclasses/superclass
+  in the dataset. If None, it is automatically set to be the size (in terms
+  of subclasses) of the smallest superclass. 
+- :samp:`split`: If ``None``, subclasses of a superclass are returned 
+  as is, without partitioning them into the source and target domains. 
+  Else, can be ``rand/good/bad`` depending on whether the subclass split should be
+  random or less/more adversarially chosen [STM20]_.
+- :samp:`ancestor`: If a node ID is specified, superclasses are chosen from 
+  subtree of class hierarchy rooted at this node. Else, if None, :samp:`ancestor`
+  is set to be the root node.
+- :samp:`balanced`: If True, subclasses/superclass is fixed over superclasses.
+
+For instance, we could create a balanced dataset, with the subclass partition 
+being less adversarial as follows:
+
+.. code-block:: python
+
+   ret = DG.get_superclasses(level=2, 
+                          Nsubclasses=None, 
+                          split="rand", 
+                          ancestor=None, 
+                          balanced=True)
+  subclass_ranges, label_map, subclass_tuple, superclasses, _ = ret                                    
+
+This method returns:
+
+- :samp:`superclasses` is a list containing the IDs of all the
+  superclasses.
+- :samp:`label_map` is a dictionary mapping a superclass
+  number (label) to name. 
+- :samp:`subclass_ranges` is a list, which for
+  each superclass, contains a list of subclasses included (in both
+  domains). 
+- :samp:`subclass_tuple` is a tuple of subclass ranges for
+  the source and target domains. For instance,
+  :samp:`subclass_tuple[0]` is a list, which for each superclass,
+  contains a list of subclasses present in the source domain.
+
+You can experiment with these parameters to create datasets of different
+granularity. For instance, you could specify the :samp:`Nsubclasses` to
+restrict the size of every superclass in the dataset,
+set the :samp:`ancestor` to be a specific node (e.g., ``n00004258`` 
+to focus on living things), or set :samp:`balanced` to ``False`` 
+to get an imbalanced dataset.
+
+we can take a closer look at the composition of the dataset---what
+superclasses/subclasses it contains---using:
+
+.. code-block:: python
+  from robustness.tools.breeds_helpers import print_dataset_info
+
+  print_dataset_info(subclass_ranges, 
+                    label_map, 
+                    subclass_tuple, 
+                    superclasses, 
+                    hier.LEAF_NUM_TO_NAME)
+
+Finally, for the source and target domains, we can create datasets
+and their corresponding loaders:
+
+.. code-block:: python
+  from robustness.datasets import DATASETS
+  # For the source domain
+  dataset_source = DATASETS['custom_imagenet'](data_dir, subclass_tuple[0])
```

Comment:
should just do (although as noted above I think the name of subclass_tuple should change):
```python
from robustness import datasets
train_subclasses, test_subclasses = subclass_tuple
dataset_source = datasets.CustomImageNet(data_dir, train_subclasses)
```